ggshield/cmd/utils/utils.py

```diff
@@ -0,0 +1,14 @@
+import re
+from pathlib import Path
+from typing import Set, Union
+
+from click import UsageError
+
+from ggshield.utils.files import is_filepath_excluded
+
+
+def assert_directory_not_ignored(
+    directory: Union[str, Path], exclusion_regexes: Set[re.Pattern]
+):
+    if is_filepath_excluded(directory, exclusion_regexes):
```

Comment:
If we scan a folder whose subfolders are all ignored, do we want to display a UsageError ? If yes, does this work with this code ?

Example with this architecture :
files_to_scan
   /tests
   /src

Our `paths_ignore` contains `tests/` and `src/,` and we scan `files_to_scan/`

Reply:
I think that according to the issue we do not want to display a UsageError.